docs/about-us.rst

```diff
@@ -0,0 +1,61 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _about-us:
+
+********
+About Us
+********
+
+``meson-python`` is a project that extends the Meson_ build system, making it
+possible to it for Python_ packaging. It is 100% open source and is available
```

Comment:
> "making it possible to it for Python_ packaging": something went wrong with this sentence.

Yes, thank you for noticing. I'll fix it.

> Also, this repeats what's in the index page. I think that anyone looking at this page would have an idea of what meson-python is, or they would not care either way.

This is supposed to be an expanded description of the project, which is relevant for people other than just the ones using it. For eg. I have received inquiries from people overseeing government software projects regarding my open source projects being used as a dependency, and having a page like this can either avoid them contacting us, or provide us a good page to point them to. This is just an example though, there are many other people that would be reading this page.

I don't really don't see a significant benefit removing the section, only the opposite, so I'd rather keep this section if this is not something you feel strongly about.

Reply:
I don't want the section removed. Just the repetition of what meson-python is. This is not an "extended description", it is an (half) sentence that is not enough to understand what the project does to who does not know it already, and is useless repetition for all others.